moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -209,10 +256,16 @@ const ProjectFundingTable = () => {
   };
 
   const LookupAutocompleteComponent = (props) => {
-    const { id, value, field } = props;
+    const { id, value, field, hasFocus } = props;
     const apiRef = useGridApiContext();
     const ref = React.useRef(null);
 
+    React.useEffect(() => {
```

Comment:
Really glad that the documented way of doing this worked out. Refs get very confusing in MUI so thanks for unraveling them! 🙏

Reply:
the lightbulb that went off was when i figured out to pass the ref to inputRef instead of ref!